testgrid/config.yaml

```diff
@@ -2138,6 +2141,9 @@ dashboards:
   - name: kind, master (dev) [non-serial]
     description: Runs conformance tests using kubetest against latest kubernetes master with a kubernetes-in-docker cluster, skipping [Serial] tests
     test_group_name: ci-kubernetes-kind-conformance-parallel
+  - name: kind, v1.15 (dev)
+    description: Runs conformance tests using kubetest against latest kubernetes release-1.15 with a kubernetes-in-docker cluster
+    test_group_name: ci-kubernetes-kind-conformance-latest-1-15
```

Comment:
for kubeadm we were told that new prow jobs should use annotations:
https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration

which are actually great, because you don't need a testgrid approver for your changes.
but will defer to @BenTheElder if he's ok with the new rules. maybe a seperate PR can move the kind jobs to annotations.

Reply:
i think we have to do this now

Reply:
admittedly I'm behind on this and test-infra in general